packages/analytics-js-integrations/src/integrations/FacebookPixel/utils.js

```diff
@@ -298,6 +315,9 @@ const getProductListViewedEventParams = properties => {
       quantity: 1,
     });
     contentType = 'product_group';
+  } else {
+    // Default to 'product' when no contentIds and no category
+    contentType = 'product';
```

Comment:
this will change the existing behaviour - ideally if no contents/products are being sent, it's invalid event & shouldn't get considered as `product` content type